### ISSUE TITLE
assets-db option status unclear

### ISSUE THREAD

**Issue by yoxteam** [uxbox/uxbox#3541]
### Steps To Reproduce

There is a discrepancy in the documentation:

* https://github.com/penpot/penpot-docs/blob/9b46d07f9be68c5d15d2f2efcdd1f9131b710478/technical-guide/developer/subsystems/assets-storage.md?plain=1#L21
  * shows the `assets-db` as an option.

* https://github.com/penpot/penpot-docs/blob/9b46d07f9be68c5d15d2f2efcdd1f9131b710478/technical-guide/configuration.md?plain=1#L295
  * says "Currently there are three backends available" and then lists only two.

The commit in which "db" is dropped seems to be only about formatting the documentation, so it might actually be dropped from the Configuration Guide by mistake:
https://github.com/penpot/penpot-docs/commit/f677fee2494e08ec24cffc119ba6b28b75b72c09#diff-a5fa29120dd3a682584aead54a908d2ff23175919ad59afd2bb92841f0fdd5a7L280-R294

### Expected behavior

3 = 3

### Actual behavior

3 = 2 ;)

### Additional context

Basically I would really like to use the assets-db option on an on-prem offline network. This ensures that our database backup strategy also covers the assets and no timing issues can occur between our separate file based backup and database backup processes.

So the main question for me is: is the assets-db option still a viable option? Or was it dropped from the documentation due to some known-issues?

@niwinz Mentioning you as you were involved in mentioned commit :-)
@myfunnyandy Mentioning you as a question with regards to what is / is not supported might be a PO question.

**Comment by niwinz** [comment]
assets-db is not available since 3 or 4 versions, so the documentation is wrong (outdated)

**Comment by niwinz** [comment]
the assets-db has a good idea for a very small instance, because there are no efficient way to transfer data from database to serve it as static file for to be consumed from the browser. This is why that backend was removed, because it provides just one benefit  but many drawbacks

**Comment by yoxteam** [comment]
Thanks for the clear and quick response!

For reference to future readers : this is the commit that removed the implementation:
https://github.com/penpot/penpot/commit/ebcb385593a1710a71ea34fa846ede4a8d5e5f94

**Comment by yoxteam** [comment]
@niwinz Now both the https://github.com/penpot/penpot-docs/issues/147 as well as this issue is closed, but the documentation is still mismatching reality. Should one of these issues be reopened?
